### DISCUSSION TITLE
Rubocop inline hints

### DISCUSSION THREAD

**Discussion by pepito2k** [zed-industries/zed#8538] [Feature Requests]
Hi, I would love to switch from VScode to Zed, but one of the things I mostly use is the inline hints from rubocop
For example:
![image](https://github.com/zed-industries/zed/assets/613343/13a1bcdd-95fd-40d8-b08d-db15ea35a60c)

Is there any chance we can get the same in Zed?

Thanks!

**Comment by Moshyfawn** [comment]
If you're looking for the error lens experince, there's a feature request in #4901.

**Reply by pepito2k** [reply]
Thanks, I looked into the discussions but didn't know the exact words for it. I'll follow that feature request :)
